src/transactions/types.ts

```diff
@@ -144,7 +144,7 @@ export type InputMultiSigDataWithRemoteABI = {
  */
 export type InputScriptData = {
   bytecode: HexInput;
-  typeArguments?: Array<TypeTag>;
+  typeArguments?: Array<TypeTag | string>;
```

Comment:
can we use a safer type here? like `${string}::${string}::${string}` and whatever format is accepted? `string` feels too weak

Reply:
It will be converted to a TypeTag, so if it's not formatted correctly it will be rejected.

Oh, and we can't use that type because `u8` is a valid TypeTag